scripts/check-update.sh

```diff
@@ -37,19 +37,8 @@ else
 fi
 
 if [ "$CURRENT_VERSION" != "$NPM_VERSION" ]; then
-    echo "UPDATING to $NPM_VERSION"
-
-    NEXT_EXTENSION_VERSION=$(node scripts/extension-version.js "$NPM_VERSION" "$EXTENSION_VERSION")
-    echo "NEXT_EXTENSION_VERSION: $NEXT_EXTENSION_VERSION"
```

Comment:
Tested this file manually and it seems to reach the correct `set-output` code path, the only thing I miss is that previously it used to print the next extension version early. 

Maybe we can bring back this section of code for log purposes.

Reply:
indeed, I'll bring that back in